.ahoy.yml

```diff
@@ -16,15 +16,15 @@ commands:
 
   cli:
     usage: Start a shell inside TEST container.
-    cmd: docker-compose exec test bash
+    cmd: docker-compose exec cli bash
```

Comment:
I thought this was done a while ago - had to do this to stop some CI build errors in gitlab for some reason the connection between test > mysql was failing.